frontend/src/bundles/main/routes.ts

```diff
@@ -0,0 +1,43 @@
+import { RouteConfig } from 'vue-router';
+const LayoutPage = () => import('./pages/LayoutPage.vue').then(m => m.default);
+const ListPage = () => import('./pages/ListPage.vue').then(m => m.default);
+const MatchPage = () => import('./pages/MatchPage.vue').then(m => m.default);
+const MapPage = () => import('./pages/MapPage.vue').then(m => m.default);
+const ChatPage = () => import('./pages/ChatPage.vue').then(m => m.default);
+const EventsPage = () => import('./pages/EventsPage.vue').then(m => m.default);
+
+export function getMainRoutes(): RouteConfig[] {
+  return [
+    {
+      component: LayoutPage,
+      path: '/',
```

Comment:
We now have the main landing on "/". Maybe you should change this route to "/user"? What do you think @AlexeyTryapko ?

Reply:
add redirect to path /list
`
redirect: {
  name: 'main-list',
},
`